src/app/store/actions/ReferenceData.actions.ts

```diff
@@ -0,0 +1,36 @@
+import { Action } from '@ngrx/store';
```

Comment:
please use this  update

`import { Action } from '@ngrx/store';
import { Preparer } from '@app/models/preparer';
import { TestStation } from '@app/models/test-station';

export enum ReferenceDataActions {
  LoadPreparers = '[Test Result Component] LoadPreparers',
  LoadTestStations = '[Test Container] LoadTestStations',
  LoadPreparersSuccess = '[Test Result Component] LoadTestStationsSuccess',
  LoadTestStationsSuccess = '[Test Container] LoadTestStationsSuccess'
}

export class LoadPreparers implements Action {
  public readonly type = ReferenceDataActions.LoadPreparers;
}

export class LoadPreparersSuccess implements Action {
  public readonly type = ReferenceDataActions.LoadPreparersSuccess;
  constructor(public preparers: Preparer[]) {}
}

export class LoadTestStations implements Action {
  readonly type = ReferenceDataActions.LoadTestStations;
}

export class LoadTestStationsSuccess implements Action {
  readonly type = ReferenceDataActions.LoadTestStationsSuccess;
  constructor(public testStations: TestStation[]) {}
}

export type ReferenceDataAction =
  | LoadPreparers
  | LoadTestStations
  | LoadPreparersSuccess
  | LoadTestStationsSuccess;`